lib/miq_expression.rb

```diff
@@ -1334,6 +1334,7 @@ def self.get_col_type(field)
 
     model = determine_model(model, parts)
     return nil if model.nil?
+    return MiqExpression::Field.parse("CustomAttribute-#{col}").column_type if col.include?(CustomAttributeMixin::CUSTOM_ATTRIBUTES_PREFIX)
```

Comment:
@alongoldboim I guess that you need to attach model of field so 

MiqExpression::Field.parse("#{model}-#{col}") and probably it is 'field' variable as is @imtayadeway is saying